docs/Using-Fleet/FAQ.md

```diff
@@ -363,3 +363,7 @@ A host is considered online if it has connected to Fleet in the last 80 (20+60)
 
 **Offline** hosts won't respond to a live query. These hosts may be shut down, asleep, or not connected to the internet.
 A host could also be offline if there is a connection issue between the osquery agent running in the host and Fleet (see [What should I do if my computer is showing up as an offline host?](#what-should-i-do-if-my-computer-is-showing-up-as-an-offline-host)).
+
+## Why aren't "additional queries" being applied to hosts enrolled in a team?
+
+Changes were introduced in Fleet v4.20.0 that caused the `additional_queries` set in Global Agent Options to no longer be included in each team's Agent Options. If you would like those queries to be applied to all hosts, they will need to be included in each team's [configuration](./configuration-files.md#team-settings) as well. 
```

Comment:
```suggestion
Changes were introduced in Fleet v4.20.0 that caused the `features.additional_queries` set in at the global level to no longer apply to hosts assigned to a team. If you would like those queries to be applied to hosts assigned to a team, you will need to be include these queries under `features.additional_queries` in each team's [configuration](./configuration-files.md#teams). 
```

Reply:
It looks like the additional queries feature is configured outside of agent options. See the example `team` YAML document here: https://fleetdm.com/docs/using-fleet/configuration-files#teams

`features` and `agent_options` are on the same level.

@ksatter can you please add this to the top of the 4.20.0 release notes? I think it's important to let folks who are upgrading to 4.20.0 and higher know about this breaking change.

Reply:
Thanks @noahtalerman! Changes committed.